Release/machine/SIRIUS/TS.V04.01/sirius_ts_lattice.m

```diff
@@ -0,0 +1,294 @@
+function [r, lattice_title, IniCond] = sirius_ts_lattice(varargin)
+% 2013-11-26 created Ximenes (from V100)
+% 2016-09-28 V01.02 - new version (Ximenes) - see 'VERSIONS.txt' in Release/machine/SIRIUS
+
+%% global parameters 
+%  =================
+
+d2r = pi/180;
+
+% --- system parameters ---
+energy = 3e9;
+lattice_version = 'TS.V04.01';
+mode = 'M';
+version = '1';
+mode_version = [mode version];
+
+% processamento de input (energia e modo de operacao)
+for i=1:length(varargin)
+    if ischar(varargin{i})
+        mode_version = varargin{i};
+    else
+        energy = varargin{i} * 1e9;
+    end;
+end
+
+lattice_title = [lattice_version '.' mode_version];
+fprintf(['   Loading lattice ' lattice_title ' - ' num2str(energy/1e9) ' GeV' '\n']);
+
+% carrega forcas dos imas de acordo com modo de operacao
+IniCond = struct('ElemIndex',1,'Spos',0,'ClosedOrbit',[0;0;0;0],'mu',[0,0]);
+set_parameters_ts;
+
+
+%% passmethods
+%  ===========
+
+bend_pass_method = 'BndMPoleSymplectic4Pass';
+quad_pass_method = 'StrMPoleSymplectic4Pass';
+
+
+%% elements
+%  ========
+% --- drift spaces ---
+ldif     = 0.1442; 
+l015     = drift('l015',  0.15, 'DriftPass');
+l020     = drift('l020',  0.20, 'DriftPass');
+l025     = drift('l025',  0.25, 'DriftPass');
+l060     = drift('l060',  0.60, 'DriftPass');
+l080     = drift('l080',  0.80, 'DriftPass');
+l090     = drift('l090',  0.90, 'DriftPass');
+l160     = drift('l160',  1.60, 'DriftPass');
+l280     = drift('l280',  2.80, 'DriftPass');
+l400     = drift('l400',  4.00, 'DriftPass');
+la2p     = drift('la2p', 0.08323, 'DriftPass');
+la3p     = drift('la3p', 0.232-ldif, 'DriftPass');
+lb1p     = drift('lb1p', 0.220-ldif, 'DriftPass');
+lb2p     = drift('lb2p', 0.83251, 'DriftPass');
+lb3p     = drift('lb3p', 0.30049, 'DriftPass');
+lb4p     = drift('lb4p', 0.19897-ldif, 'DriftPass');
+lc1p     = drift('lc1p', 0.18704-ldif, 'DriftPass');
+lc2p     = drift('lc2p', 0.07304, 'DriftPass');
+lc3p     = drift('lc3p', 0.19934, 'DriftPass');
+lc4p     = drift('lc4p', 0.72666-ldif, 'DriftPass');
+ld1p     = drift('ld1p', 0.25700-ldif, 'DriftPass');
+ld2p     = drift('ld2p', 0.05389, 'DriftPass');
+ld3p     = drift('ld3p', 0.154, 'DriftPass');
+ld4p     = drift('ld4p', 0.192, 'DriftPass');
+ld5p     = drift('ld5p', 0.456, 'DriftPass');
+ld6p     = drift('ld6p', 0.258, 'DriftPass');
+ld7p     = drift('ld7p', 0.175, 'DriftPass');
+
+% --- markers ---
+start    = marker('start',  'IdentityPass');
+fim      = marker('end',    'IdentityPass');
+
+% --- beam screen monitors
+scrn   = marker('Scrn', 'IdentityPass');
+
+% --- beam current monitors ---
+ict    = marker('ICT', 'IdentityPass');
+fct    = marker('FCT', 'IdentityPass');
+
+% --- beam position monitors ---
+bpm    = marker('BPM', 'IdentityPass');
+
+% --- correctors ---
+ch     = corrector('CH',  0, [0 0], 'CorrectorPass');
+cv     = corrector('CV',  0, [0 0], 'CorrectorPass');

```

Comment:
Same comment as in lnls-fac/pymodels#25.

Reply:
done